### PR TITLE
Edit(IDP-1203): Move Extension Tutorial

### PR DESCRIPTION
**Pull request by MIA-Deltat1995**
## Pull Request Type

- [x] Documentation content changes
- [ ] Bugfix / Missing Redirects
- [ ] Docusaurus site code changes

## PR Checklist

- [x] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [x] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [x] No sensitive content has been committed

### CODE REVIEW

**Review by fredmaggiowski — CHANGES_REQUESTED** [review]

File: docs/console/tutorials/create-extension.md
[the remark this answers is not in the document]
Comment (akelity) [reply docs/console/tutorials/create-extension.md:1]: +1

[the remark this answers is not in the document]
Comment (MIA-Deltat1995) [reply docs/console/tutorials/create-extension.md:1]: done

**Review by akelity — APPROVED** [review]
LGTM

**Review by fredmaggiowski — APPROVED** [review]
LGTM

**Pull request merged**